docs/src/content/docs/de/reference/overrides.md

```diff
@@ -18,59 +20,73 @@ Um deine eigenen Komponenten zu schreiben, importiere den `Props`-Typ von Starli
 
 ```astro
 ---
+// src/components/Custom.astro
 import type { Props } from '@astrojs/starlight/props';
 
-const { hatSeitennavigation } = Astro.props;
+const { hasSidebar } = Astro.props;
 //      ^ Typ: boolean
 ---
 ```
 
 So erhaltest du die Autovervollständigung und Angabe des Datentyps beim Zugriff auf `Astro.props`.
 
-### Props
+### Eigenschaften
 
 Starlight wird die folgenden Props an deine eigenen Komponenten übergeben.
 
 #### `dir`
 
-**Type:** `'ltr' | 'rtl'`
+**Typ:** `'ltr' | 'rtl'`
 
 Schreibrichtung der Seite.
 
 #### `lang`
 
-**Type:** `string`
+**Typ:** `string`
 
 BCP-47-Sprachkennzeichen für das Gebietsschema dieser Seite, z.B. `de`, `zh-CN` oder `pt-BR`.
 
 #### `locale`
 
-**Type:** `string | undefined`
+**Typ:** `string | undefined`
 
 Der Basispfad, unter dem eine Sprache angeboten wird. `undefined` für Root-Locale-Slugs.
 
+#### `siteTitle`
```

Comment:
I cannot suggest an edit for it as it's outside of the review range but this change is missing the anchor change from https://github.com/withastro/starlight/pull/1620/files#diff-ad80506c0d18072cbd9a85e2090c2be41e5a5523fe9f2991a1c653e41d089591, the `-1` addition to properly link to the component and not the prop.

Reply:
I got u 😊